packages/generator/src/typedoc.ts

```diff
@@ -50,21 +50,32 @@ export function getPropertyDescription(
   property: VdmProperty,
   constraints: VdmPropertyValueConstraints = { nullable: false }
 ): string {
-  return addConstraints(
-    property.description ||
-      endWithDot(toTitleFormat(property.instancePropertyName).trim()),
-    constraints
+  return makeBlockComment(
```

Comment:
I like the approach of having one function that creates valid input for block comments. In this case I initially thought that `makeBlockComment` actually creates something that looks like this:
```
/**
 * desc
 */
```
which it doesn't. It actually creates a block comment ready description. Maybe this can be reflected in the name of the function. Realistically what it does is preparing a block comment description or prepending `\n`.

Reply:
Second minor point: consider whether `makeBlockComment` shouldn't be called outside of `getPropertyDescription`. It seems as if this was a little bit more consistent that way. But as I said, minor.